.travis.yml

```diff
@@ -13,13 +13,13 @@ notifications:
     on_failure: always
     on_pull_requests: false
 node_js:
-- 8.15.1
+- 12.18.3
```

Comment:
`semver@7.3.2` requires a node version `>=10` and `get-caller-file@2.0.5` requires a node version  `6.* || 8.* || >= 10.*`.

Perhaps I could downgrade the version of these packages to avoid it, but it didn't feel unreasonable to upgrade to the latest LTS version of Node. If anybody says this could cause other issues, I'll downgrade the version of these packages.

Reply:
should be fine since gulp has been upgraded lately!